backend/app/api/v1/website.go

```diff
@@ -785,7 +785,7 @@ func (b *BaseApi) GetDirConfig(c *gin.Context) {
 // @Summary Get default html
 // @Accept json
 // @Param type path string true "type"
-// @Success 200 {object} response.FileInfo
+// @Success 200 {object} response.WebsiteHtmlRes
 // @Security ApiKeyAuth
 // @Security Timestamp
 // @Router /websites/default/html/{type} [get]
```

Comment:
The code changes look generally appropriate and follow standard API documentation conventions, but there are some slight inconsistencies:

1. **Status Code Mismatch**: The `@Success` annotations for methods that return arrays or objects (like `List website names`, `Get site permissions`, etc.) should use `{array}` instead of just `string`. For instance:
   ```go
   // @Success 200 {array} response.WebsiteOption
   ```

2. **Response Objects**: Some responses might not explicitly define their response structure using types like `response.NginxRewriteRes`, `response.WebsiteProxyConfig` etc. While it's generally okay to omit defining a specific type if it's straightforward and clear what data is expected, it can be beneficial to define custom structs to ensure consistency and clarity.

3. **File Info Object**: Replace `{object}` with `{object} response.FileInfo` in the `/websites/default/html/{type}` method since `FileInfo` represents file information which needs a more precise response definition.

These adjustments will improve readability and maintainability of your API endpoint definitions.